ufm-test-services/grafana/dashboards/metamask.json

```diff
@@ -0,0 +1,182 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": {
+          "type": "grafana",
+          "uid": "-- Grafana --"
+        },
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
```

Comment:
Should we have some share const where we pull different colors, so that is easy to configure a new  annotation?

Reply:
This would be very nice if grafana supoprts this

Reply:
I don't think we need to do anything fancy with this file as it's generated by Grafana. This color is the default red that I selected to be used by the metrics through the Grafana dashboard editor UI. So, any metric that chooses to use red will default to use the same color